cmd/auth-provider-gcp/apis/credentialprovider/README.md

```diff
@@ -0,0 +1 @@
+Temporarily copied from code that will land in k8s v1.20 - please reach out to SIG Cloud Provider with any questions.
```

Comment:
The 1.20 ship has sailed. The earliest this code can land in K/K is 1.21.

Reply:
This is from https://github.com/kubernetes/kubernetes/pull/94196 - I added a link to the PR in the README, but could there be a better approach?

Reply:
Marking as resolved since I unfortunately made it sound like this code hasn't merged yet (it's in 1.20, but cloud-provider-gcp doesn't support 1.20 yet, I believe - the README also reflects this now), sorry for the confusion.